### ISSUE TITLE
complete() menu is incorrectly positioned

### ISSUE THREAD

**Issue by xzfc** [neovim/neovim#9140]
- `nvim --version`: v0.3.1
- Vim 8.1 behaves differently
- Operating system/version: Arch Linux
- Terminal name/version: XTerm(337)
- `$TERM`: xterm

### Steps to reproduce:

`issue.vim`:
```vim
set nocompatible
inoremap <C-l> <C-r>=Complete()<CR>
set splitbelow
function! Complete()
  call complete(col('.'), range(100))
  return ''
endfunction

ped /dev/null
for i in range(10)
  call append(0, i)
endfor
startinsert
```

* `nvim -u issue.vim`
* press `<C-l>`

### Actual behaviour

Completion menu is placed outside the current window and it is not adjacent to the current line.
![neovim screenshot](http://dump.bitcheese.net/files/suwamox/nvim.png)

### Expected behaviour

See vim screenshot:
![vim screenshot](http://dump.bitcheese.net/files/kunomys/vim.png)

### Affected use-case
This issue makes `deoplete.nvim` preview window unusable in case if it is located at the bottom of the screen.
![neovim screenshot](http://dump.bitcheese.net/files/zypapud/nvim-deoplete.png)

**Comment by justinmk** [comment]
This was probably addressed by https://github.com/neovim/neovim/pull/8913 and https://github.com/neovim/neovim/pull/8996

**Comment by xzfc** [comment]
It is fixed on `v0.3.2-722-g1cf50cbfd`.

**Comment by justinmk** [comment]
That's usually a good time to close the issue, right?
